packages/schemas/src/models/tenants.ts

```diff
@@ -8,7 +8,7 @@ export const Tenants = createModel(/* sql */ `
     id varchar(21) not null,
     db_user varchar(128),
     db_user_password varchar(128),
-    name varchar(128) not null,
+    name varchar(128) not null default 'My Project',
```

Comment:
should we add an issue to drop this the default in the next major release? @xiaoyijun has created a linear project

Reply:
LOG-6228 added to track this breaking change.